src/os/bluestore/BlueFS.cc

```diff
@@ -2229,6 +2241,143 @@ int64_t BlueFS::_read_random(
   return ret;
 }
 
+void BlueFS::_wal_update_size(FileRef file) {
```

Comment:
I think size for WALv2 files is not updated.
_replay is starting checking wal files from 0:
`
2024-05-10T12:11:29.476+0000 7fe5bea18b40 20 bluefs _replay 0x10df5:  op_dir_link  db.wal/000288.log to 281
2024-05-10T12:11:29.476+0000 7fe5bea18b40 10 bluefs _read h 0x55752ad43a40 0x0~8 from file(ino 281 size 0x0 mtime 2024-05-10T12:00:39.045747+0000 allocated 2340000 alloc_commit 2340000 extents [1:0xdf3c000~11a0000,1:0x1d19ed3000~11a0000] type WAL_V2)
`

Reply:
Because we don't rely on `fnode.size`, we don't even care that is the value of `fnode.size` from disk reads, we just update it by reading range of 0x0~allocated_length